CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 # Change Log
 All notable changes to the "azure-appservice" extension will be documented in this file.
 
+## 0.7.0 - 2018-04-05
+### Added
+- Subscription filter button next to Subscription nodes in the explorer
+- Deploy to Web App context menu action for Web Apps
+- Open in Portal context menu action for Subscriptions
```

Comment:
just my opinion, but I don't think open in portal for subscriptions is worth including